src/controllers/tripsController.js

```diff
@@ -179,11 +182,41 @@ export default class requestController {
         email,
         status: 'Open',
         stops,
-        returnDate
+        returnDate,
+        managerId: user.managerId
       });
       return Response.success(res, 201, res.__('Multi city request created successfully'), newMulticityRequest);
     } catch (error) {
       return Response.errorResponse(res, 500, error.message);
     }
   }
+
+  /**
+   * @param  {object} req
+   * @param  {object} res
+   * @return {object} confirmed request
+   */
+  static async confirmRequest(req, res) {
+    const { user } = req;
```

Comment:
Could we use `try and catch` to catch any errors?

Reply:
I forgot them but I just added them